Forms/Schema/FormSchema.php

```diff
@@ -67,10 +67,10 @@ public function getState(Form $form) {
 			$this->getFieldStates($form->Actions())
 		);
 
-		if($form->Message()) {
-			$state['messages'][] = [
```

Comment:
I know the PHP `Form` API can only show a single message at the moment, but I'm not sure we should carry through this limitation to `FormSchema` and frontend code as well. I can easily imagine situations where you might want different form messages, e.g. one styled as an error, the other one as a warning. Can we just use an array with a single element instead?